src/collection-preferences/utils.tsx

```diff
@@ -133,6 +137,73 @@ export const ContentDensityPreference = ({ label, description, value, onChange }
   </InternalCheckbox>
 );
 
+interface StickyColumnsPreferenceProps extends CollectionPreferencesProps.StickyColumnsPreference {
+  onChange: (value: any) => void;
+  value?: any;
+}
+
+interface StickyPreference extends CollectionPreferencesProps.StickyColumnsPreference {
+  onChange: (value: any) => void;
+  preference: {
+    title: string;
+    description: string;
+    options: ReadonlyArray<{
+      label: string;
+      value: number;
+    }>;
+  };
+  value?: any;
+  firstOrLast: 'first' | 'last';
```

Comment:
why this one can not be bool value like `isFirst`?

Reply:
Even if a bit more verbose, I would prefer to keep it in line with our external API, and having it at as a string allows us to use it directly for creating the class name.

Reply:
@rubencarvalho make sense, thank you for clarification